build.sbt

```diff
@@ -40,7 +40,8 @@ lazy val root = (project in file("."))
       "org.scalatest" %% "scalatest" % "3.0.5" % "it,test",
       "junit" % "junit" % "4.13.2" % "it,test",
       "org.testcontainers" % "mysql" % "1.15.1" % "it,test",
-      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.12.3"
+      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.12.3",
+      "org.projectlombok" % "lombok" % "1.18.20" % "provided"
```

Comment:
Since we are adding lombok, perhaps we can use @Getter/@Setter in the POJO classes and delete the ones we generated.

Reply:
deleted all generated getter and setter method